util/util.go

```diff
@@ -1,46 +1,46 @@
 package util
 
 import (
-	"hash/crc32"
-	"io"
-	l "log"
```

Comment:
Why did I use log instead of fmt last time. Is `log` something I should be using to log errors?

Reply:
log is useful for creating structured and unstructured logs that will probably go into a file. fmt is good for printing to the command line.

Actually, this reminds me that I should record logs on the user's system in case something goes wrong... #69